packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -31828,16 +31828,16 @@ Map {
   "DashboardIcon" => Object {},
   "DataScientistIcon" => Object {},
   "DataTableIcon" => Object {},
-  "Emptystate404Icon" => Object {},
-  "EmptystateDefaultIcon" => Object {},
-  "EmptystateErrorIcon" => Object {},
-  "EmptystateNoresultsIcon" => Object {},
-  "EmptystateNotauthorizedIcon" => Object {},
-  "EmptystateSuccessIcon" => Object {},
   "ImageIcon" => Object {},
   "LineGraphIcon" => Object {},
   "ListIcon" => Object {},
   "SimpleBarGraphIcon" => Object {},
   "ValueKpiIcon" => Object {},
+  "EmptyStateDefaultIcon" => Object {},
+  "EmptyState404Icon" => Object {},
+  "EmptyStateErrorIcon" => Object {},
+  "EmptyStateNoResultsIcon" => Object {},
+  "EmptyStateNotAuthorizedIcon" => Object {},
+  "EmptyStateSuccessIcon" => Object {},
```

Comment:
Do we want to match the previous casing? 
ie. `EmptystateNoresultsIcon` instead of `EmptyStateNoResultsIcon`

Reply:
Yeah I suppose I should. This casing make my head hurt.

Reply:
heard that.